gnd/src/main/res/layout/fragment_terms.xml

```diff
@@ -86,8 +106,7 @@
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
- />
-
-
+      app:onButtonClick="@{__ -> viewModel.onTermsAccepted()}"
```

Comment:
I haven't written android code in a few years. Please check with style within the app to ensure that binding within xml is ok and consistent.

Reply:
Data and event binding is now a standard feature of Android Jetpack:
https://developer.android.com/topic/libraries/data-binding